packages/components/molecules/f-rating/src/components/Rating.vue

```diff
@@ -111,44 +133,67 @@ export default {
         getRatingStarPercentage () {
             return `${(this.starRating / this.maxStarRating) * 100}%`;
         }
+    },
+
+    methods: {
+        /**
+         * Display review rating message format.
+         *
+         * @todo - If the component is using `short` as a `starRatingSize` we shouldn't display text
+         * alongside it for now. (TBC with design - ticket in backlog).
+         *
+         * @returns {string|boolean}
+         */
+        getRatingDisplayFormat () {
+            return this.ratingDisplayType
+                ?
+                this.$t(`ratings.ratingDisplayType.${this.ratingDisplayType}`, {
+                    rating: this.starRating,
+                    maxStarRating: this.maxStarRating,
+                    totalReviews: this.totalReviews
+                })
+                : false;
+        }
     }
 };
 </script>
 
 <style lang="scss" module>
 @use '@justeat/fozzie/src/scss/fozzie' as f;
 
-.c-rating-starWrapper {
+.c-rating {
+    display: flex;
+}
+
+.c-rating-stars {
     margin: 0;
     padding: 0;
     position: relative;
     display: inline-block;
 }
-    .c-rating-container {
+    .c-rating-stars-icons {
         display: flex;
     }
 
     .c-rating-star {
-        &--small {
-            width: 12px;
-        }
+        width: 12px;
```

Comment:
feel these should be variables at the top so they could be easily changed, more ideally / out of scope for this is that icon component has a size parameter.

Reply:
There's a ticket in our backlog about sizing so will be tackled there :)

Reply:
I agree with this :)